### ISSUE TITLE
Why is LSTM prediction is so slow in tfjs-node?

### ISSUE THREAD

**Issue by betances** [tensorflow/tfjs#3284]
Hello! 

Im trying to make predictions of this model developed in python and passed to tfjs-node:

`In1 = Input(shape=(100, 100), name="Input1")
lstm1, forward_h, forward_c, backward_h, backward_c = Bidirectional(LSTM(100, return_sequences=True, return_state=True), merge_mode="concat", name="bi1")(In1)

In2 = Input(shape=(100, 100), name="Input2")
lstma = Bidirectional(LSTM(100, return_sequences=False),merge_mode="concat", name="bi2")(In2, initial_state=[forward_h, forward_c, backward_h, backward_c])

Out = Dense(100, activation="softmax",name="dense1")(lstma)

model = Model(inputs=[In1, In2], outputs=[Out])

tfjs.converters.save_keras_model(model, 'ances_model')`

But when I make a prediction of single example in tfjs-node:

`
x1=tf.zeros([1,100,100])
x2=tf.zeros([1,100,100])

a=model.predict([x1,x2])`

This is very slow (just under 1 seconds for get prediction in a i7). 

And it is very curious because, a 5 million parameter CNN takes a few milliseconds to make a prediction!

Any idea what could be improved?

I have already verified that I am using tfjs-node instead of tfjs, and I have tried to reduce the amount of parameters, and the quantization.

**Comment by pyu10055** [comment]
@betances If you are using the model only for prediction, you should save the model to saved model format and use tfjs-node direct execution. 
https://blog.tensorflow.org/2020/01/run-tensorflow-savedmodel-in-nodejs-directly-without-conversion.html

**Comment by betances** [comment]
> @betances If you are using the model only for prediction, you should save the model to saved model format and use tfjs-node direct execution.
> https://blog.tensorflow.org/2020/01/run-tensorflow-savedmodel-in-nodejs-directly-without-conversion.html

You have saved my life. :)

Thank you
